### PR TITLE
Rename wrapped_error and make it private

### PR DESCRIPTION
**Pull request by AdRiley**
### Pull Request Description

Rename wrapped_error to DB_wrapped_error and make it private to avoid name clashes as part of https://github.com/enso-org/enso/issues/8981

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [x] The documentation has been updated, if necessary.
- [ ] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [x] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- All code has been tested:
  - [x] Unit tests have been written where possible.
  - [ ] If GUI codebase was changed, the GUI was tested when built using `./run ide build`.

### CODE REVIEW

**Review by GregoryTravis — APPROVED** [review]

File: distribution/lib/Standard/Database/0.0.0-dev/src/Internal/DB_Wrapped_Error.enso [hunk 1]
```diff
@@ -0,0 +1,6 @@
+private
+
+## PRIVATE
+type DB_Wrapped_Error
+    ## PRIVATE
+    Value value
```
Comment (radeusgd) [inline distribution/lib/Standard/Database/0.0.0-dev/src/Internal/DB_Wrapped_Error.enso]:
```suggestion
    Value value

```
We usually try to end our files with a newline.

If you are using IntelliJ, I recommend these settings:
image

(I often forget about this too 😅)

**Review by radeusgd — APPROVED** [review]

File: distribution/lib/Standard/Database/0.0.0-dev/src/Internal/DB_Wrapped_Error.enso (hunk 1 above)
Comment (AdRiley) [reply distribution/lib/Standard/Database/0.0.0-dev/src/Internal/DB_Wrapped_Error.enso]: Ack that's the second time I have done that :(. Will find the equivalent in VSCode.

Comment (AdRiley) [reply distribution/lib/Standard/Database/0.0.0-dev/src/Internal/DB_Wrapped_Error.enso]: Fixed and set in my VSCode settings so I won't do it again. Thanks for the tip.

Comment (AdRiley) [reply distribution/lib/Standard/Database/0.0.0-dev/src/Internal/DB_Wrapped_Error.enso]: Did that work? It looks right locally but not in GitHub...

Comment (radeusgd) [reply distribution/lib/Standard/Database/0.0.0-dev/src/Internal/DB_Wrapped_Error.enso]: > Did that work? It looks right locally but not in GitHub...

What did not work in GitHub?

Comment (AdRiley) [reply distribution/lib/Standard/Database/0.0.0-dev/src/Internal/DB_Wrapped_Error.enso]: image

I expected to see a line 7

Comment (radeusgd) [reply distribution/lib/Standard/Database/0.0.0-dev/src/Internal/DB_Wrapped_Error.enso]: Ah, no the last line is not shown if empty - but it no longer shows the 'No newline at end of file warning'  so it looks good

**Pull request merged**